README.md

```diff
@@ -12,14 +12,53 @@ Here you'll find starter code and more advanced example use cases.
 </div>
 
 
-## Recipes 🍯
+## Recipes: Getting Started 🍯
+#### Subflows
+- [Getting Started With Subflows](https://discourse.prefect.io/t/migrating-to-prefect-2-0-from-flow-of-flows-to-subflows/1318)
```

Comment:
I think it's ok to link to Discourse posts for now, but someday perhaps migrate code from Discourse to the directories in this repo so that we can maintain them better with version control, and upon breaking changes in the core library, add commits that reference the valid Prefect version?

Reply:
It's funny you mention maintainability because I originally added the flows into `flows-starter/` before instead referencing the Discourse articles in the `README`.. because inevitably I was linking the code back to the Discourse to provide more information. The idea of moving the code from Discourse to the repo and the Discourse tutorial (non-code portions) into a `README` seems.. well I'm unsure on it. Right now, I know Anna is updating Discourse posts she has created, as changes happen/new features lead to better ways of doing things. It feels too chaotic and Sisyphean to try to mirror that in a non-automated way to me, right now. Do you visualize the solution looking different than how I described?

Reply:
@anna-geller Do you have thoughts on this?

Reply:
I see! Below are some of my random ideas and suggestions, with emphasis on "suggest":

I don't think all the Discourse posts need to transferred over necessarily, only the popular ones, and once they are transferred over to GitHub, we make sure it's polished and follows the best practice / standards (so maybe we should dump a pre-commit file in this repo https://github.com/PrefectHQ/prefect-collection-template/blob/main/%7B%7Bcookiecutter.collection_name%7D%7D/.pre-commit-config.yaml).

Also, perhaps the Discourse posts can be converted to notebooks (which support markdown + code), and
eventually, have these notebooks displayed to docs pages?

As an example from my poorly maintained package, I have unexecuted notebooks in the docs directory, https://github.com/ahuang11/ahlive/blob/main/docs/source/introductions/quick_start.ipynb, and when I push a new version, readthedocs executes the notebooks and publishes them https://ahlive.readthedocs.io/en/latest/essentials/referencing.html. Unexecuted notebooks are useful in the sense that every time the docs are building, it'll crash if there's a breaking change. Notably, some examples like devops/infrastructure cannot be demonstrated in a notebook.